metsim/disaggregate.py

```diff
@@ -6,90 +6,103 @@
 import pandas as pd
 
 import metsim
-from metsim.defaults import PARAMS as params
-from metsim.defaults import CONSTS as consts 
+from metsim.configuration import PARAMS as params
+from metsim.configuration import CONSTS as consts
 
-tiny_rad_fract = np.zeros(366) #This is updated during the mtclim run
 
-def disaggregate(df_daily):
+def disaggregate(df_daily, solar_geom):
     """
     TODO
     """
-    dates_hourly = pd.date_range(metsim.start, metsim.stop, freq='H') 
+    end = metsim.stop + pd.Timedelta('1 days')
+    dates_hourly = pd.date_range(metsim.start, end, freq='H')
```

Comment:
I made it so that timestep is specified in mintutes.  Maybe at a later date we can look at contextualizing between hours and minutes, but that's purely convenience so I'm not focusing on it currently.